openmc/material.py

```diff
@@ -973,3 +973,73 @@ def export_to_xml(self, path='materials.xml'):
         # Write the XML Tree to the materials.xml file
         tree = ET.ElementTree(root_element)
         tree.write(path, xml_declaration=True, encoding='utf-8')
+
+
+def make_boric_acid(boron_ppm, temperature=293., pressure=0.1013, density=None,
```

Comment:
Would "boric acid" more accurately be referred to herein as "borated water"?

Reply:
I imagine that boric acid is the only way "borated water" is made in practice, but I guess that name gives a better idea of how the internal calculation is done.  I'll change it to "borated water"